plugins/module_utils/openvpn_server.py

```diff
@@ -316,6 +313,13 @@ def _create_target(self):
     def _find_target(self):
         """ find the XML target_elt """
         (target_elt, self.idx) = self._find_openvpn_server(self.obj['description'])
+        for param in ['shared_key', 'tls']:
+            before_value = self.pfsense.get_element(param, target_elt)
+            if before_value is None and self.params[param] == 'generate':
```

Comment:
could you try it with an `or` here, again?

Reply:
wait i understand, we should skip it

Reply:
i hope, that i solved it now

Reply:
That looks better now.